platform/security/common/src/main/java/ddf/security/samlp/impl/SamlValidator.java

```diff
@@ -287,6 +288,20 @@ private Request(Builder builder) {
       logoutRequest = (LogoutRequest) builder.xmlObject;
     }
 
+    /**
+     * A SAML LogoutRequest must include at least one SessionIndex, if the logout is initiated by
+     * Session Participant. However, if the Session Authority initiates the logout, a SessionIndex
+     * is optional. In that case, the absence of a SessionIndex means to logout of all sessions
+     * associated with the Principle.
+     *
+     * @throws ValidationException
+     */
+    protected void checkSessionIndexes() throws ValidationException {
+      if (logoutRequest.getSessionIndexes().isEmpty()) {
```

Comment:
❓ So since we are always throwing the exception if sessionIndexes is empty, Do we never have the case mentioned above where the session authority initiates the logout?

Reply:
We do not support session authority initiated logouts. 
At least not yet.
I think I might feel better removing the validation. While it is currently correct, I can imagine thing changing in other parts of security that would make empty session indexes either possible, or at least not a fatal error.

Reply:
fixed